src/MeshBVH.js

```diff
@@ -167,17 +168,55 @@ export default class MeshBVH {
 		const roots = [];
 		const ranges = this._getRootIndexRanges( geo );
 
-		for ( let range of ranges ) {
+		if ( ranges.length === 1 ) {
 
 			const root = new MeshBVHNode();
-			root.boundingData = ctx.getBounds( range.offset, range.count, new Float32Array( 6 ) );
+			const range = ranges[ 0 ];
+
+			if ( geo.boundingBox != null ) {
+
+				root.boundingData = boxToArray( geo.boundingBox );
+
+			} else {
+
+				root.boundingData = ctx.getBounds( range.offset, range.count, new Float32Array( 6 ) );
+
+			}
+
 			splitNode( root, range.offset, range.count );
 			roots.push( root );
 
-			if ( reachedMaxDepth && options.verbose ) {
+		} else {
+
+			for ( let range of ranges ) {
+
+				const root = new MeshBVHNode();
+				root.boundingData = ctx.getBounds( range.offset, range.count, new Float32Array( 6 ) );
+				splitNode( root, range.offset, range.count );
+				roots.push( root );
+
+				if ( reachedMaxDepth && options.verbose ) {
```

Comment:
It looks like these statements won't get logged if there's only one geometry group. I think this can actually be moved outside of the loop to before the return statement, which should fix that. I don't think there's a reason for this to be printed multiple times when creating a new BVH.